website/docs/r/manifest.html.markdown

```diff
@@ -200,6 +200,7 @@ The following arguments are supported:
 #### Arguments
 
 - **rollout** (Optional) When set to `true` will wait for the resource to roll out, equivalent to `kubectl rollout status`. 
+- **conditions** (Optional) A set of conditions to wait for.
```

Comment:
The `conditions` block has two nested attributes `status` and `type`. Could you please mention them in the documentation too?

One small comment regarding style here. Could you please make all argument names formatted as `code` instead of **bold text**?